Tools/CISupport/ews-build/steps.py

```diff
@@ -7352,6 +7352,95 @@ def getResultSummary(self):
         return {u'step': status}
 
 
+class DownloadUnexpectedResultsFromMaster(transfer.FileDownload):
+    mastersrc = WithProperties('public_html/results/%(buildername)s')
```

Comment:
Does this need to be updated given the changes to https://github.com/WebKit/WebKit/pull/36340/files#diff-32440cf8b81d39efcd9ad9036d550d72daef76fbff0f30dc3e0f85a415896990R7354?